apps/super-app/src/services/ViewServices.ts

```diff
@@ -45,6 +54,25 @@ export interface FacebookLink {
   text: string;
 }
 
+export function getServiceCardAbouts(): ServiceCardAbout[] {
```

Comment:
@glencbz technically this is not a "common" function as it's specific to the Service Cards.